src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java

```diff
@@ -122,6 +124,10 @@ public ClientLauncher() {
 
       ComponentHelper.centreOnScreen(this);
       setVisible(true);
+      JRootPane rootPane = SwingUtilities.getRootPane(launchButton); 
+      rootPane.setDefaultButton(launchButton);
+      //getRootPane().setDefaultButton(launchButton);
```

Comment:
delete this comment.

Reply:
Oh yeah, I should try to get that to work properly :)

Reply:
Now sets focus to Launch button on window open event if a server is selected